Guesh 07 bnf abb 114

GueshSol opened the pull request.

thea-m commented on ParisBNF/abb/BNFabb114.xml:
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                              corresp="AT1116MMJerusalem"/> typically in four episodes/scenes,
+                           showing from the bottom left to the right: first, the sick pilgrim lying
+                           in bed; second the sick pilgrim standing before the places to which <persName ref="PRS6819Mary"/>
+                           carries him; third, in varying sequence Golgotha, Jerusalem, the mount of
```
the same is true for "varying" as for "typically": This only makes sense in the art theme, here, there must be one concrete sequence

thea-m commented on ParisBNF/abb/BNFabb114.xml:
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                     <decoNote type="miniature" xml:id="d22">
+                        <locus target="#50v"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AT1127MMQatir"/>typically with sequence of three
```
corresp="AT1127MMQatir"/>typically  space missing

thea-m commented on ParisBNF/abb/BNFabb114.xml:
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AT1130MMMarqorewos"/>typically with sequence of seven
+                           episodes/events, showing: first, the leprous bishop Marqorewos standing
+                           before Patriarch Zakārǝzās and showing him his leprosy; second, the
```
Zakārǝyās

thea-m commented on ParisBNF/abb/BNFabb114.xml:
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                           Marqorewos praying before the icon of the <ref type="authFile"
+                              corresp="AT1031VirginAndChild"/>; fourth, <ref type="authFile"
+                                 corresp="AT1031VirginAndChild"/> healing the leprous bishop and performing gesture of blessing; fifth,
+                           the bishop showing himself healed to patriarch Zakārǝzās; sixth, the bishop showing himself healed to his student ; and
```
Zakārǝyās

thea-m commented on ParisBNF/abb/BNFabb114.xml:
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                        <locus from="65v" to="66r"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AT1132MMSophia"/>typically with sequence of seven
+                           episodes/events, showing: first, Sofyā lying on top of a building before
```
decide whether you want to use Sophia or Sofyā in the description

thea-m commented on ParisBNF/abb/BNFabb114.xml:
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                           episodes/events, showing: first, ʾAnǝsṭāsyos standing and praying in a church
+                           where the icon of the <ref type="authFile" corresp="AT1031VirginAndChild"/> depicted in front of him; second,ʾAnǝsṭāsyos lying
+                           before the icon of <ref type="authFile" corresp="AT1031VirginAndChild"/>, who is extending her hand towards him and announcing him good news
+                           cncerning his illness.
```
concerning

thea-m commented on ParisBNF/abb/BNFabb114.xml:
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AAT1135MMSalusi"/> typically with sequence of four
+                           episodes/events, showing: first, the monk Sālusi surrounded by other monks,
+                           who are attacking him; second, the monks preventing him for eating;
```
preventing him to eat?

thea-m commented on ParisBNF/abb/BNFabb114.xml:
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                        <desc> A miniature of the art theme <ref type="authFile"
+                              corresp="AT1136MMCannibal"/> typically with sequence of eight
+                           episodes/events, showing: first, the cannibal surrounded by the bodies of
+                           those he killed; Second the cannibal asking the peasant to sell his ox;
```
second

thea-m commented on ParisBNF/abb/BNFabb114.xml:
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                           episodes/events, showing: first, the widow's son stealing cloth from a
+                           church; second, members of the Church arresting and starting to punish
+                           the thief; third, <persName ref="PRS6819Mary"/> ordering the punishers to
+                           let him free performing gesture of blessing; and fourth, the thief
```
a gesture of blessing, here and throughout

thea-m commented on ParisBNF/abb/BNFabb114.xml:
```diff
@@ -355,584 +360,987 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                     </decoNote>
+                     <decoNote type="miniature" xml:id="d34">
+                        <locus target="#80v"/>
+                        <desc> A miniature of the art theme <ref type="authFile"
```
It might make sense to state that these are depicted from bottom to top?

thea-m reviewed (changes requested):
Very good, Guesh! I made some comments regarding typos. 
And a general comment: I think that you copied the description from the Art Themes, which is fine, but be careful that the Art Themes describe the topic as an abstract, to be valid for all individual renderings it might receive in manuscripts. Here, you describe the concrete rendering of the Art Theme in this manuscript, so instead of describing what the miniatures look like "typically" you can describe how they actually look like here.

GueshSol commented:
Thank you @thea-m for your excellent remarks, yes I copied the sequence of description from the art themes and just adjusted them how the decorations are distributed in my manuscript in reality, and except for the first phrases after the authority files of the decorations that I added ''typically..'' the rest sequences are arranged in on the basis of the miniatures in the MS.

thea-m commented on ParisBNF/abb/BNFabb114.xml:
```diff
@@ -913,406 +912,417 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                              corresp="AT1031VirginAndChild"/> and second, <ref type="authFile"
-                              corresp="AT1031VirginAndChild"/> performing a gesture of blessing up
-                           on the peasant. 
+                              corresp="AT1109MMWorker"/> with one two episodes, showing: first, the
```
"one two episodes" - is it one or two? :)

thea-m commented on ParisBNF/abb/BNFabb114.xml:
```diff
@@ -913,406 +912,417 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                              corresp="AT1116MMJerusalem"/> with four episodes, showing, first, the
+                           sick pilgrim lying in bed; second the sick pilgrim standing before the
+                           places to which <persName ref="PRS6819Mary"/> carries him; third, in
+                           varying sequence Golgotha, Jerusalem, the mount of Olives, Bethlehem and
```
please delete "in varying sequence"

thea-m commented on ParisBNF/abb/BNFabb114.xml:
```diff
@@ -913,406 +912,417 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                        <desc> These decorations are stylistically different from the illustrative
+                           miniatures and probaly could be additions by another atrist at some time
+                           after the completion of the first hand textual and decorative compostion
+                           of the amnsucript. They contain images of several saints, martyrs and
```
manuscript :)

thea-m reviewed (changes requested):
Thank you, wonderful, just some last minor remarks
